src/components/map/Map.js

```diff
@@ -21,7 +14,6 @@ class Map extends Component {
   };
 
   state = {
-    bbox: null,
```

Comment:
Stops box state moved to App as well. It has to do with requesting stops, so Map shouldn't be concerned.